pkg/occlient/occlient.go

```diff
@@ -3069,6 +3069,18 @@ func (c *Client) IsVolumeAnEmptyDir(volumeMountName string, dc *appsv1.Deploymen
 	return false
 }
 
+// IsVolumeAnEmptyDir returns true if the volume is an EmptyDir, false if not
```

Comment:
Can you please update this comment? Rest seems to work for me.

Reply:
Done.